Assets/Scripts/AILookBehaviour.cs

```diff
@@ -0,0 +1,90 @@
+using System.Collections;
+using UnityEngine;
+
+public enum LookActionType
```

Comment:
This should be named be LookActionTypes. Do we need it to be public? Could get away with it being private if it's only used in 1 place.